src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -42,7 +36,7 @@ public void parseCommand_clear() throws Exception {
         assertTrue(parser.parseCommand(ClearCommand.COMMAND_WORD + " 3") instanceof ClearCommand);
     }
 
-    /* This test has been commented out as the delete and edit commands have been temporarily
+    /* This test has been commented out as the delete, edit and find commands have been temporarily
```

Comment:
Not exactly meant to be part of your PR, but we might want to shift these test cases into `ProfileCommandParserTest`. Just leaving it here to KIV.